connector/connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectPlannerSuite.scala

```diff
@@ -329,19 +346,25 @@ class SparkConnectPlannerSuite extends SparkFunSuite with SparkConnectPlanTest {
       .setAllColumnsAsKeys(true)
       .addColumnNames("test")
 
-    val e = intercept[InvalidPlanInput] {
-      transform(proto.Relation.newBuilder.setDeduplicate(deduplicate).build())
-    }
-    assert(
-      e.getMessage.contains("Cannot deduplicate on both all columns and a subset of columns"))
+    checkError(
+      exception = intercept[InvalidPlanInput] {
+        transform(proto.Relation.newBuilder.setDeduplicate(deduplicate).build())
+      },
+      errorClass = "CONNECT.INVALID_PLAN_INPUT",
+      parameters = Map("msg" -> "Cannot deduplicate on both all columns and a subset of columns"))
```

Comment:
Would it possible to move all the plain text to `error-classes.json`. Could you image that we will translate errors to local languages in the future, or tech editors will fix some typos in the texts. In your case, we will have to rebuild Spark which is not nice approach.

Reply:
Happy to move the literals out.